src/test/functional/pages/BuzzPage.ts

```diff
@@ -1,20 +1,30 @@
 import { BasePage } from "./BasePage";
+
 export class BuzzPage extends BasePage {
     protected readonly sharePhotosButton = this.page.getByRole("button", { name: "Share Photos" })
     protected readonly fileInput = this.page.locator('input[type="file"]');
     protected readonly submitPhoto = this.page.getByRole("button", { name: 'Share', exact: true })
```

Comment:
you use it for VIDEO and PHOTO sharing so I propose to change the name for univerasal one

Reply:
what is your proposition then?

Reply:
shareButton?

Reply:
i will change to sharingSubmitButton